diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt

```diff
@@ -66,9 +66,12 @@ class PackageNaming(configRules: List<RulesConfig>) : DiktatRule(
                 // calculating package name based on the directory where the file is placed
                 val realPackageName = calculateRealPackageName(filePath, configuration)
 
+                // if the file path contains "buildSrc" - don't add the package name to the file
+                val isNotBuildSrcPath = "buildSrc" in filePath
```

Comment:
```suggestion
                val isBuildSrcPath = "buildSrc" in filePath
```
Also, I think we can move this check inside `warnAndFixMissingPackageName`, because it affects only missing package names